cmd/transactioncmd/transaction_commit.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package transactioncmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var txFilePath string
+
+// avalanche transaction commit
+func newTransactionCommitCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "commit [subnetName]",
```

Comment:
if the tx is CreateBlockchain, we need subnet to update sidecar, if the tx is AddValidator, no need to update

Reply:
maybe the unique argument (non flag) should be the path to the signed tx

Reply:
I think it becomes inconsistent with all the other commands. Everywhere else we use the `[subneName]` as a non-flag command argument. Even if not needed for `AddValidator`, it might be good for the user to nevertheless require the subnet name, so that it is very clear what the user is doing.

Therefore I vote to keep the subnet name as a mandatory argument.